src/actions/segment/segment.ts

```diff
@@ -28,6 +44,61 @@ export class SegmentAction extends Hub.Action {
   supportedVisualizationFormattings = [Hub.ActionVisualizationFormatting.Noapply]
   requiredFields = [{ any_tag: this.allowedTags }]
 
+  taggedFields(fields: any, tags: string[]) {
+    return fields.filter((f: any) =>
+      f.tags && f.tags.some((t: string) => tags.indexOf(t) !== -1),
+    )
+  }
+
+  taggedField(fields: any, tags: string[]) {
+    return this.taggedFields(fields, tags)[0]
+  }
+
+  segmentFields(fields: any): SegmentFields {
```

Comment:
if we import `lookml_model_explore_field` using the fancy api types importer thing and `import LookmlModelExploreField as Field` we could actually type these with real types instead of `any`